core/pkg/server/server.go

```diff
@@ -137,6 +140,29 @@ func (s *Server) Serve(portFile string) error {
 	return nil
 }
 
+// makeListener starts listening on a socket and returns its connection info.
+func (s *Server) makeListener() (net.Listener, PortInfo, error) {
+	listener, portInfo, err := MakeUnixListener(s.parentPID)
```

Comment:
why is the unix listener not a target function on the server like the tcp one?

Reply:
could be just a wrapper for consistency

Reply:
created a `server/listeners` subpackage and moved both functions there